roles/debian-packaging/tasks/main.yaml

```diff
@@ -50,3 +51,12 @@
     state: present
   become: yes
   when: pkgs.stat.isdir is defined and pkgs.stat.isdir
+
+- name: Install packages
+  # shell: "piuparts -d buster --allow-database --warn-on-debsums-errors --extra-repo='deb [trusted=yes] http://mirror.wazo.community/debian/ wazo-dev-buster main' ../*.deb"
```

Comment:
@fredericlepied There are any reason why we need to keep this commented line? I don't see any explanation in the commit message or in the comment itself.